app/helpers/application_helper/toolbar/middleware_messaging_center.rb

```diff
@@ -0,0 +1,36 @@
+# noinspection RubyArgCount
+class ApplicationHelper::Toolbar::MiddlewareMessagingCenter < ApplicationHelper::Toolbar::Basic
+  button_group('middleware_messaging_monitoring', [
+    select(
+      :middleware_messaging_monitoring_choice,
+      'product product-monitoring fa-lg',
+      t = N_('Monitoring'),
```

Comment:
it is correct that there is a mechanism to collect metrics. But at the same time, there should be certain configurations made to specify which metrics should be collected, which is not part of this PR.